packs/tesla/aliases/honk_horn.yaml

```diff
@@ -5,6 +5,6 @@ description: "Honk your horn"
 formats:
    - display: "honk-honk {{vin}}"
      representation:
-       - "((beep|honk|meep)-?)+ {{vin}}[!.]?"
+       - "((beep|honk|meep)-?)+ {{vin}}[!.]*"
```

Comment:
vin is optional. this would mean you would have to put a space before the exclamation marks. 

`honk!!!!` wouldn't be accepted here